lib/dgram.js

```diff
@@ -256,11 +257,21 @@ Socket.prototype.sendto = function(buffer,
                                    port,
                                    address,
                                    callback) {
-  if (typeof offset !== 'number' || typeof length !== 'number')
-    throw new Error('Send takes "offset" and "length" as args 2 and 3');
+  if (typeof offset !== 'number') {
+    throw new errors.Error('ERR_INVALID_ARG_TYPE', 'offset', 'number');
```

Comment:
That should probably be a `TypeError` as well as the other errors thrown in the `sendTo` function.